pkg/apis/operator/v1alpha1/tektonpipeline_types.go

```diff
@@ -107,7 +107,18 @@ type PipelineProperties struct {
 
 	// ScopeWhenExpressionsToTask Deprecated: remove in next release
 	ScopeWhenExpressionsToTask *bool `json:"scope-when-expressions-to-task,omitempty"`
-	PipelineMetricsProperties  `json:",inline"`
+
+	EnforceNonfalsifiability  string `json:"enforce-nonfalsifiability,omitempty"`
+	EnableKeepPodOnCancel     *bool  `json:"keep-pod-on-cancel,omitempty"`
+	ResultExtractionMethod    string `json:"results-from,omitempty"`
```

Comment:
json tag looks different than the variable name, is that exoected?
`ResultExtractionMethod != results-from`

Reply:
I followed what pipeline has done and what we're doing for other fields https://github.com/tektoncd/pipeline/blob/main/pkg/apis/config/feature_flags.go#L125